ext/symengine/symengine.c

```diff
@@ -113,6 +113,10 @@ void Init_symengine() {
     c_dirichlet_eta = rb_define_class_under(m_symengine, "Dirichlet_eta", c_function);
     c_zeta = rb_define_class_under(m_symengine, "Zeta", c_function);
     c_gamma = rb_define_class_under(m_symengine, "Gamma", c_function);
+    c_one_arg_function = rb_define_class_under(m_symengine, "OneArgFunction", c_function);
+
+    //Abs Class
+    c_abs = rb_define_class_under(m_symengine, "Abs", c_function);
```

Comment:
We can add it later when an actual need arises.

It usually doesn't occur to anyone to remove something that is not being
used(as long as that doesn't obstruct anything) but it will occur to us if
we feel something is missing.

On Tue, Jun 7, 2016, 11:11 PM Rajith Vidanaarachchi <
notifications@github.com> wrote:

> In ext/symengine/symengine.c
> <https://github.com/symengine/symengine.rb/pull/50#discussion_r66117718>:
>
> > @@ -113,6 +113,10 @@ void Init_symengine() {
> >      c_dirichlet_eta = rb_define_class_under(m_symengine, "Dirichlet_eta", c_function);
> >      c_zeta = rb_define_class_under(m_symengine, "Zeta", c_function);
> >      c_gamma = rb_define_class_under(m_symengine, "Gamma", c_function);
> > +    c_one_arg_function = rb_define_class_under(m_symengine, "OneArgFunction", c_function);
> > +
> > +    //Abs Class
> > +    c_abs = rb_define_class_under(m_symengine, "Abs", c_function);
>
> I was thinking it's better to keep the same class hierarchy as in C++,
> maybe it will be useful in the future. Currently there's absolutely no
> practical use though. What do you think? I can get rid of it if you want.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/symengine/symengine.rb/pull/50/files/189089ac7e60c7ce7299f33f62fe63aa8d6bdb30#r66117718>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGmQ6xP1b6DDIkMiCA8Cypitn0pGdp8oks5qJa0zgaJpZM4IwKla>
> .
>